### DISCUSSION TITLE
AI Integration

### DISCUSSION THREAD

**Discussion by ajbott** [PullDating/Org#173] [Ideas & Features]
- AI Assistant
    - Personalities
    - Learning about what they want by asking open ended questions
    - Vibe matching
  - Upload own AI personality

Import user profile to AI. Backend AI could ask questions to help improve profile, recommend matches, discover more about the user, ect.

Early concept, but OpenAI current is working on different plugins that would give AI different personalities and help with different types of tasks. This could be implemented here.

**Comment by viktor-lesyk** [comment]
Hell yeah! @ajbott I'm happy to see we're on the same page here ;) 

Copying my comment from Discord:

> I also had some crazy ideas today, like adding an AI chatbot to the app to:
> 
> a) gather filter criteria in a more "friendly chat" way: people write their preferences in a text message and the chatbot helps to define filter values or gives additional questions
> b) gather people's thoughts, issues, and pain points with other apps and use AI to generalize and find the trends

And your answer was:

> I would love the idea of an AI chatbot to gather the filter criteria in a more personalized and friendly manner. I think in order to accomplish this first we would first have to gather the data into what filters create the most effective connections for people (what are people looking for the most, what people have filled out, what's important to the person, ect).
> 
> I think AI analysis of feedback would really help to condense down what the most general categories people struggle with would help the most to start. Understanding why the matches fail and understanding the causes of these failed connections helps us to work on fixing the reasons why things aren't working

**Comment by viktor-lesyk** [comment]
https://meetdara.ai/

from https://www.elatedate.com/

**Comment by viktor-lesyk** [comment]
https://discord.com/channels/862791338687856640/1223736674073313401/1223757706175971408

> To create a profile description, I just had to answer 3 questions, and some AI created a summary for me 🙂 not the best one, but it can certainly work. Already something to start with.

**Comment by Oshyan** [comment]
I think there are several important and distinct concepts to tease out here. At least these 3 and maybe more:
1. Using AI to actually influence matching somehow, e.g. sentiment and interest analysis of written profile text, personality profile generation based on responses to question prompts (a la OKC), etc.
2. Generating potentially complex _filter_ criteria based on easy natural language text input, e.g. "I'm looking for someone within 10 years of my age who is no taller than me and is interested in rugby and travel"
3. AI analysis of _user/app feedback_

Each of those has different areas of impact, degrees of difficulty for implementation, etc. The 3rd one is probably most accessible now. The system we had dabbled in using for project management, Fibery, actually does include some basic AI features that can be used to semi-automatically summarize feedback from various sources (email, survey, community discussion, Github, in-app feedback), as well as potentially automatic linking of user feedback to distilled issues or feature requests.
